src/openms/include/OpenMS/CHEMISTRY/ElementDB.h

```diff
@@ -124,16 +126,14 @@ namespace OpenMS
      */
     double calculateMonoWeight_(const Map<UInt, double> & Z_to_mass);
 
-    /*_ read elements from a XML file, formatted as a Param file.
-
-            @throw throws ParseError if the file cannot be parsed
-            @throw throws FileNotFound if the file could not be found
-     */
-    void readFromFile_(const String & file_name);
+	// constructs element objects
+    void storeElements();
 
+  // add element objects to documentation maps
+    void addElementToMaps(const String& name, const String& symbol, const UInt an, const Element* e);
 
-    /// store element after parsing it
-    void storeElement_(const UInt an, const String& name, const String& symbol, const Map<UInt, double>& Z_to_abundancy, const Map<UInt, double>& Z_to_mass);
+  // constructs isotope objects
+    void storeIsotopes(const String& name, const String& symbol, const UInt an, const Map<UInt, double>& Z_to_mass, const IsotopeDistribution& isotopes);
```

Comment:
if possible use std::string and std::map

Reply:
Yes, I wanted to do this next.